gcp/appengine/frontend_handlers.py

```diff
@@ -585,3 +585,24 @@ def package_in_ecosystem(package):
 def osv_has_vuln(vuln_id):
   """Checks if an osv vulnerability exists for the given ID."""
   return osv.Bug.get_by_id(vuln_id)
+
+
+@blueprint.app_template_filter('list_packages')
+def list_packages(vuln_affected: list[dict]):
```

Comment:
Can you double check if this type is dict? (e.g. printing out the type with `type(vuln_affected)`)

Reply:
type(vuln_affected) shows list, and type(affected) shows dict. So I think vulnerability.affected is a list[dict].

Reply:
ah cool, thanks!